mlos_bench/mlos_bench/tunables/tunable_groups.py

```diff
@@ -127,7 +131,8 @@ def __iter__(self):
             An iterator over all tunables in all groups. Each element is a 2-tuple
             of an instance of the Tunable parameter and covariant group it belongs to.
         """
-        return ((group.get_tunable(name), group) for (name, group) in self._index.items())
+        for (name, group) in self._index.items():
+            yield (group.get_tunable(name), group)
```

Comment:
I much prefer the original syntax. do we have to unroll it just to please mypy?

Reply:
Not necessarily, but the original syntax changes the return type from an iterator to a sequence I think which was inconsistent with the use elsewhere.